<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerLongPETCTFindingSelectionWidget</class>
 <widget class="QWidget" name="qSlicerLongPETCTFindingSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QFormLayout" name="FormLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="LabelSelectFinding">
       <property name="text">
        <string>Select Finding:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBoxFinding" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="ctkCheckBox" name="CheckBoxROIVisiblity">
         <property name="text">
          <string>ROI visibility</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="indicatorIcon" stdset="0">
          <iconset resource="../qSlicerLongPETCTModule.qrc">
           <normaloff>:/Icons/VisibleOn.png</normaloff>
           <normalon>:/Icons/VisibleOff.png</normalon>:/Icons/VisibleOn.png</iconset>
         </property>
         <property name="indicatorIconSize" stdset="0">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qmrmlnodecombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerLongPETCTModule.qrc"/>
 </resources>
 <connections/>
</ui>
